### PR TITLE
Command_oblivion.java

### PR DESCRIPTION
**Pull request by Adamx120**
"/oblivion" - Similar with the doom

Does:
- Doom MSGs
- Ban UUID
- Ban IP
- Rollback

Features:
- Super Admin Access
- Removed: Explosion, Remove SuperAdmin, Ignite

### CODE REVIEW

**Comment by OnlineRepz** [comment]
First, the issue is not accepted (You must have an accepted issue in order to make PRs)
Second, you only changed the message.
And third of all, command doom have barely used.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_oblivion.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        new BukkitRunnable()
+        {
+            @Override
+            public void run()
+            {
+                // message
+                TFM_Util.adminAction(sender.getName(), "Banning " + player.getName() + ", IP: " + ip, true);
+
+                // kick player
+                player.kickPlayer(ChatColor.RED + "FUCKOFF, and get your shit together!");
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_oblivion.java]: You may want to change this to:

player.kickPlayer(ChatColor.RED + "You have been obliterated by:" + " " + sender.getName() + "!");

**Comment by ValenciaOrange** [comment]
did you just make a command that is more powerful than doom and gave it to the superadmins so that they can doom anyone?

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_oblivion.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.util.Vector;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH, blockHostConsole = true)
```
Comment (ValenciaOrange) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_oblivion.java]: You should probably change this to senior

**Comment by LegendIsAwesomes** [comment]
By the looks of it this is just a ripoff of /doom.

**Comment by JeromSar** [comment]
This PR does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and thus is closed.

**Pull request closed without merging**